pages/shoppingCart.js

```diff
@@ -25,59 +24,146 @@ export default function ShoppingCart({ ticket }) {
     setCartTickets([]);
     setTimeout(() => {
       setShowPopup(false);
-    }, 1800);
+    }, 3000);
   };
 
   return (
     <>
-      <h2>Shopping Cart Page</h2>
-      <button onClick={goBack}>back</button>
-
+      <Header />
+      <BackButton />
       {cartTickets && cartTickets.length > 0 ? (
         <>
           <ul>
             {cartTickets.map((ticket) => (
-              <li key={ticket.id}>
-                <Image
-                  src={ticket.image}
-                  alt="Ticket"
-                  priority="high"
-                  width={"280"}
-                  height={"150"}
-                />
-                <p>{ticket.rating}</p>
-                <p>{ticket.name}</p>
-                <p>{ticket.location}</p>
-                <p>{ticket.date}</p>
-                <p>
-                  {ticket.price} {ticket.currency}
-                </p>
-                <button onClick={() => handleDeleteTicket(ticket)}>
-                  DELETE
-                </button>
-              </li>
+              <StyledLi key={ticket.id}>
+                <StyledImageDiv>
+                  <StyledImageCard
+                    src={ticket.image}
+                    alt="Ticket"
+                    priority="high"
+                    width={"550"}
+                    height={"300"}
+                  />
+                </StyledImageDiv>
+                <StyledNameDiv>
+                  <p>{ticket.name}</p>
+                </StyledNameDiv>
+                <StyledTicketInfoDiv>
+                  <p>{ticket.rating}</p>
+                  <p>{ticket.location}</p>
+                  <p>{ticket.date}</p>
+                  <p>
+                    {ticket.price} {ticket.currency}
+                  </p>
+                  <button onClick={() => handleDeleteTicket(ticket)}>
```

Comment:
Instead of passing the entire `ticket` you'd only pass down the `id` of the ticket and compare ids within the function.